tap_service_titan/streams.py

```diff
@@ -686,7 +696,89 @@ def path(self) -> str:
         return f"/accounting/v2/tenant/{self._tap.config['tenant_id']}/export/invoices"
 
 
-class EstimatesStream(ServiceTitanStream):
+class InvoiceItemsStream(ServiceTitanExportStream):
+    """Define custom stream."""
```

Comment:
I'm just noting that these docs strings are all boilerplate string, probably from the SDK (?), keeping them as the are is fine since thats the status quo but maybe we have a follow up to refresh them.